### ISSUE TITLE
AVT 2 - React Toolbar Keyboard only navigation and color contrast issues.

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#3382]
## Environment
macOS Mojave version 10.14.5
Chrome Version 75.0.3770.100 (Official Build) (64-bit)
Carbon v10 - React
Color Contrast Analyzer

## Detailed Description
1. Tab to the Seach icon
2. Press Space to open the Search text input field
3. Enter text and the X icon displays
4. Tab and the Search icon gets focus. **Expected Result:**  KB navigation to the X icon to clear the text. 
5. Press Space on the Search icon and the Search Text box closes, but the X icon is still present. (see screenshot):
<img width="298" alt="Screen Shot 2019-07-12 at 11 17 11 AM" src="https://user-images.githubusercontent.com/21676914/61143832-ea09b600-a498-11e9-9089-5d1c53dedb3f.png">

Run Color Contrast Analyzer on the X icon and Search placeholder text and they both fail WCAG color contrast requirements (see screenshots):
<img width="940" alt="Screen Shot 2019-07-12 at 11 26 09 AM" src="https://user-images.githubusercontent.com/21676914/61143923-276e4380-a499-11e9-8afe-d13a567837dc.png">
<img width="914" alt="Screen Shot 2019-07-12 at 11 30 12 AM" src="https://user-images.githubusercontent.com/21676914/61143927-29d09d80-a499-11e9-8155-1cb84a86276b.png">

**Comment by aledavila** [comment]
Issue with this is that it did not get updated to v10 styles. Input clear button is a pseudo element that doesn't allow for focus as its not in the dom.  

Proposed fix: copy data table toolbar functionality to standalone toolbar to get it up to date. This will solve the focus issue as well as the color ratio. 

Needs to be cleared with design.
